### DISCUSSION TITLE
Endpoints used by Evergreen

### DISCUSSION THREAD

**Discussion by aaronparker** [EUCPilots/evergreen-module#458] [Ideas]
I’ve added a feature to the Evergreen API that returns the list of URLs used by Evergreen to find application versions/installers, and downloads: https://stealthpuppy.com/evergreen/endpoints/. The list of URLs is important for firewall/proxy configs in restricted environments. Would love comment on output returned to see what’s returned now is OK, or whether it requires modifying

**Comment by bmitchell76** [comment]
I spent days with Fiddler tracking things to get them opened on our proxy; thanks for adding this!  I will do some testing this week.

**Comment by aaronparker** [comment]
Nice. Keen to hear whether the format it returns is enough information or whether more is required. Note all ports are TCP 80/443.

**Comment by aaronparker** [comment]
Also note that I've changed `/endpoints/updates` to `/endpoints/versions`

**Reply by bmitchell76** [reply]
The formatting is good; in our org, we can use this to make the appropriate proxy/firewall requests.  The only problem is on the proxy side; they sometimes balk at opening up a top-level domain and want to filter down to child folders.
